### PR TITLE
v13 fix

### PR DESCRIPTION
**Pull request by RedSparr0w**
**Please describe the changes this PR makes and why it should be merged:**
Updated to work with Discord.js v13

As far as I can tell from my limited testing, this seems to allow it to work with Discord.js v13+

**Status and versioning classification:**
- This PR includes breaking changes (methods removed or renamed, parameters moved or removed)

### CODE REVIEW

**Comment by RedSparr0w** [comment]
As far as I could tell everything was working as it should although I haven't used this before so I may have just not noticed some things.

**Comment by TeoxModding** [comment]
> As far as I could tell everything was working as it should although I haven't used this before so I may have just not noticed some things.

I think is need to Change the Message.channel.send(`here`, embed) to Message.channel.send({content: `Here`, embeds: [embed]})

That Was One of the Things That Changed About V13
https://discord.js.org/#/docs/main/stable/class/TextChannel?scrollTo=send

**Review by Scraayp — CHANGES_REQUESTED** [review]
Please include the changes made to discord.js v13 with: 

- https://discord.js.org/#/docs/main/stable/typedef/ChannelType
- https://discordjs.guide/additional-info/changes-in-v13.html#sending-messages-embeds-files-etc
- The rest I will do my self in the v.6.0 pr

**Comment by RedSparr0w** [comment]
> > As far as I could tell everything was working as it should although I haven't used this before so I may have just not noticed some things.
> 
> I think is need to Change the Message.channel.send(`here`, embed) to Message.channel.send({content: `Here`, embeds: [embed]})
> 
> That Was One of the Things That Changed About V13
> [discord.js.org/#/docs/main/stable/class/TextChannel?scrollTo=send](https://discord.js.org/#/docs/main/stable/class/TextChannel?scrollTo=send)

I think it only needs the `{ content: 'my message' }` if there is more than just the message AFAIK.
But I've updated it to use content anyway, and `embeds: [embed]`.
Also updated channel type check.
_Should_ all be fine now 🤞

**Comment by Scraayp** [comment]
Now please test it. If it's all fine let me know.

**Review by Scraayp — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -54,17 +54,17 @@ const { EventEmitter } = require('events')
...
  * @property {number} [warnThreshold=3] Amount of messages sent in a row that will cause a warning.
+ * @property {number} [muteThreshold=4] Amount of messages sent in a row that will cause a mute.
  * @property {number} [kickThreshold=5] Amount of messages sent in a row that will cause a kick.
- * @property {number} [muteThreshold=7] Amount of messages sent in a row that will cause a mute.
- * @property {number} [banThreshold=4] Amount of messages sent in a row that will cause a ban.
+ * @property {number} [banThreshold=7] Amount of messages sent in a row that will cause a ban.
  *
  * @property {number} [maxInterval=2000] Amount of time (ms) in which messages are considered spam.
- * @property {number} [maxDuplicatesInterval=2000] Amount of time (ms) in which duplicate messages are considered spam.
+ * @property {number} [maxDuplicatesInterval=10000] Amount of time (ms) in which duplicate messages are considered spam.
```
Comment (Scraayp) [inline index.js:62]: I would like to keep this as 2000 for now!

File: index.js [hunk 2]
```diff
@@ -140,17 +140,17 @@ class AntiSpamClient extends EventEmitter {
...
 
 			warnThreshold: options.warnThreshold || 3,
+			muteThreshold: options.muteThreshold || 4,
 			kickThreshold: options.kickThreshold || 5,
 			banThreshold: options.banThreshold || 7,
-			muteThreshold: options.muteThreshold || 4,
 
 			maxInterval: options.maxInterval || 2000,
-			maxDuplicatesInterval: options.maxDuplicatesInterval || 2000,
+			maxDuplicatesInterval: options.maxDuplicatesInterval || 10000,
```
Comment (Scraayp) [inline index.js:148]: I would like to keep this as 2000!

**Review by Scraayp — CHANGES_REQUESTED** [review]
Also please keep this at 2000 for now! Or atleast explain why it should be 10000!

**Review by Scraayp — APPROVED** [review]

**Comment by PuneetGopinath** [comment]
Hello all

A small notice:
All SomethingID has been changed to SomethingId in v13
And when exploring the code I saw it checks if `message.guild.ownerID === message.author.id` where ownerID is no longer a property in djs v13

_If you want me to open a pr, let me know_

**Comment by Scraayp** [comment]
I will add it to the 2.6.0 update thanks

**Pull request merged**